### ISSUE TITLE
Switch Pro controller doesn't register analog inputs all the way to 100%

### ISSUE THREAD

**Issue by thomas0465** [Hydr8gon/VitaControl#13]
I'm using a switch pro controller, when I test with VitaTester the analog sticks on the controller only go to 30-40 (out of 0-255), while the analog sticks on the vita work on the full range.

**Comment by dodokoki** [comment]
yeah I'm having the same problem, the analog sticks go to 230-233 out of 255 which makes, at least for me, for example Danganronpa 2 unplayable (the cursor moves sluggishly, the screen can't be moved at all sometimes)

**Comment by MrCraft112** [comment]
same for me, i cannot use it to run on minecraft. I used vivid to connect my switch controller to my ps3 through the psvita, and the character walking doesn't run and it is extremely infuriating. is there any way to deal with it on our own?
